code/modules/fabrication/designs/general/designs_general.dm

```diff
@@ -106,3 +106,11 @@
 		/decl/material/solid/glass =   CEILING((SHEET_MATERIAL_AMOUNT * FABRICATOR_EXTRA_COST_FACTOR)/2),
 		/decl/material/solid/plastic = CEILING((SHEET_MATERIAL_AMOUNT * FABRICATOR_EXTRA_COST_FACTOR)/2)
 	)
+
+/datum/fabricator_recipe/struts
+	path = /obj/item/stack/material/strut/mapped/steel
+
+/datum/fabricator_recipe/fiberglass/get_resources()
```

Comment:
`fiberglass`
Also IMO it would be good to add a recipe for aluminium and titanium struts to the industrial fab (will need to add titanium to the base_storage_capacity list though).